.ci/Jenkinsfile

```diff
@@ -207,79 +143,6 @@ pipeline {
         }
       }
     }
-    stage('Release') {
-      options { skipDefaultCheckout() }
-      environment {
-        HOME = '/var/lib/jenkins'
-      }
-      when {
-        beforeAgent true
-        tag pattern: 'v\\d+.*', comparator: 'REGEXP'
-      }
-      stages {
-        stage('Notify') {
-          steps {
-            notifyStatus(slackStatus: 'warning', subject: "[${env.REPO}] Release *${env.TAG_NAME}* ready to be pushed",
-                         body: "(<${env.RUN_DISPLAY_URL}|Open>).\n Changes: ${env.TAG_NAME}")
-          }
-        }
-        stage('Release') {
-          steps {
-            deleteDir()
-            unstash 'source'
-            prepareRelease(){
-              sh 'rake release'
-            }
-          }
-        }
-        stage('Update Branch') {
-          steps {
-            deleteDir()
-            unstash 'source'
-            prepareRelease(){
-              catchError(buildResult: 'SUCCESS', message: 'Update branch task failed', stageResult: 'UNSTABLE') {
-                sh 'rake release:update_branch'
-              }
-            }
-          }
-          post {
-            success {
-              notifyStatus(slackStatus: 'good', subject: "[${env.REPO}] Release *${env.TAG_NAME}* published",
-                           body: "Great news, the release has been done successfully. (<${env.RUN_DISPLAY_URL}|Open>).")
-            }
-            unsuccessful {
-              notifyStatus(slackStatus: 'warning', subject: "[${env.REPO}] Release *${env.TAG_NAME}* has some sync-up post-release failures.",
-                           body: "Please go to ${env.BUILD_URL} to review the logs. Most likely you need to update the branch manually. (<${env.RUN_DISPLAY_URL}|Open>)")
-            }
-          }
-        }
-        stage('Opbeans') {
```

Comment:
Tag creation is missed in https://github.com/elastic/opbeans-ruby/tree/main/.github

Reply:
This will be added here: https://github.com/elastic/opbeans-ruby/pull/171/files